packages/ui/src/forms/Form/index.tsx

```diff
@@ -277,14 +274,10 @@ export const Form: React.FC<FormProps> = (props) => {
 
       try {
         let res
-        const actionEndpoint =
-          actionArg ||
-          (typeof action === 'string'
-            ? `${action}${qs.stringify(formQueryParams, { addQueryPrefix: true })}`
-            : null)
+        const actionEndpoint = typeof actionArg === 'string' ? actionArg : null
 
         if (actionEndpoint) {
```

Comment:
This could likely be simplified, you can just check the typeof `actionArg` as the if statement, and then use it as the fetch endpoint.

Reply:
This way you do not need to create the `actionEndpoint` arg.